### PR TITLE
Breadcrumbs

### PR DESCRIPTION
**Pull request by montychristo**
## Changes in this PR

- Added gem `loaf` to allow us to define and maintain a trail of breadcrumb links across the application with a consistent syntax: https://github.com/piotrmurach/loaf
- Configured breadcrumb navigation as per https://trello.com/c/xksfSld3/226-easier-navigation-using-breadcrumbs 

## Screenshots of UI changes

*BEFORE:*
Screenshot 2021-09-09 at 15 16 53

*AFTER:*
Screenshot 2021-09-09 at 15 17 31

### CODE REVIEW

File: spec/features/self-serve/dashboard/existing_journey_spec.rb [hunk 1]
```diff
@@ -19,6 +19,11 @@
         visit "/dashboard"
       end
 
+      it "dashboard.breadcrumbs" do
+        assert_equal page.all("li.govuk-breadcrumbs__list-item").collect(&:text),
```
Comment (peterdavidhamilton) [inline spec/features/self-serve/dashboard/existing_journey_spec.rb:23]: We should maintain consistency and use `expect`

**Comment by peterdavidhamilton** [comment]
@montychristo is loaf used on any other Gov/DfE projects?

**Comment by montychristo** [comment]
> @montychristo is loaf used on any other Gov/DfE projects?

Yes, found it here:
https://github.com/DFE-Digital/academy-transfers-frontend
https://github.com/DFE-Digital/get-into-teaching-app
https://github.com/DFE-Digital/npd-find-and-explore

**Review by EdwinKruglov — APPROVED** [review]

**Review by AndrewCowle — APPROVED** [review]

**Review by peterdavidhamilton — APPROVED** [review]
Would be great to come back and get a custom matcher created for asserting breadcrumbs with clean DSL. Please drop a TODO in there and feel free to tackle it at your convenience. 👍🏼

**Pull request merged**